api/notd/block_processor.py

```diff
@@ -122,12 +114,12 @@ async def _merge_erc1155_transfers(self, erc1155Transfers: List[RetrievedTokenTr
             firstTransfer = firstTransfers.get(transferKer)
             if firstTransfer:
                 firstTransfer.amount += transfer.amount
+                firstTransfer.value += transfer.value
             else:
                 firstTransfers[transferKer] = transfer
         return list(firstTransfers.values())
 
-    async def _process_erc1155_batch_event(self, event: LogReceipt, blockData: BlockData) -> List[RetrievedTokenTransfer]:
-        blockNumber = blockData['number']
+    async def _process_erc1155_batch_event(self, event: LogReceipt,) -> List[RetrievedTokenTransfer]:
         transactionHash = event['transactionHash'].hex()
         registryAddress = chain_util.normalize_address(event['address'])
         logging.debug(f'------------- {transactionHash} ------------')
```

Comment:
you removed this above, either remove it everywhere or keep it everywhere. (i think remove unless you actually use it)

Reply:
Sorry this isn't clear to me

Reply:
this logging line was removed from the other function.